pandas/tests/test_multilevel.py

```diff
@@ -2410,6 +2410,14 @@ def test_repeat(self):
         m_df = pd.Series(data, index=m_idx)
         assert m_df.repeat(3).shape == (3 * len(data), )
 
+    def test_rangeindex_fallback_coercion_bug(self):
```

Comment:
actually I think this would be betterin ``pandas/tests/indexes/test_multi.py`` as it directly affects that.